presto-spi/src/main/java/io/prestosql/spi/function/FixedSizeWindowFunction.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.spi.function;
+
+public interface FixedSizeWindowFunction
+{
+    int[] selectPositions(int positionCount);
```

Comment:
selectedPosition?

Reply:
well, it selects positions by calling this. caller might not know whether it selects at this point or returns selected-positions.

Reply:
OK. Understood